test/Microsoft.AspNet.Mvc.Core.Test/Rendering/DefaultTemplatesUtilities.cs

```diff
@@ -44,9 +44,9 @@ public class ObjectWithScaffoldColumn
             public string Property3 { get; set; }
         }
 
-        public static HtmlHelper<ObjectTemplateModel> GetHtmlHelper()
+        public static HtmlHelper<ObjectTemplateModel> GetHtmlHelper(string idAttributeDotReplacement = null)
```

Comment:
Not a fan at all of introducing this new ambiguity in the (numerous) `GetHtmlHelper()` overloads.  Callers using `GetHtmlHelper("some string")` should expect the Model to be "some string".

In addition we've successfully avoided using optional parameters throughout this class.

Instead add another (unambiguous) `GetHtmlHelper<TModel>(Something something, string idAttributeDotReplacement)` overload and in the longest overload.  Call the longest overload directly rather than adding optional parameters anywhere along the line.